win32ss/printing/base/spoolss/main.c

```diff
@@ -228,8 +228,15 @@ SplInitializeWinSpoolDrv(PVOID* pTable)
     // Verify that all calls succeeded.
     for (i = 0; i < 9; i++)
         if (!pTable[i])
+            FreeLibrary(hWinspool);
             return FALSE;
 
+    if (hWinspool)
+    {
+        // Unload the driver module
+        FreeLibrary(hWinspool);
```

Comment:
This is wrong. The function pointers will stop being valid if the library is unloaded.